plugins/source/azure/resources/services/compute/virtual_machine_assess_patches.go

```diff
@@ -0,0 +1,72 @@
+package compute
+
+import (
+	"context"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v4"
+	"github.com/cloudquery/cloudquery/plugins/source/azure/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func virtualMachineAssessPatches() *schema.Table {
+	tableName := `azure_compute_virtual_machine_assess_patches`
+	return &schema.Table{
+		Name:     tableName,
+		Resolver: fetchVirtualMachineAssessPatches,
+		Description: `https://learn.microsoft.com/en-us/rest/api/compute/virtual-machines/assess-patches?tabs=HTTP#virtualmachineassesspatchesresult.
+
+This will begin patch assessments on available virtual machines and can take long to complete.
+
+Not available for all VMs. More at https://learn.microsoft.com/en-us/azure/virtual-machines/automatic-vm-guest-patching#requirements-for-enabling-automatic-vm-guest-patching
+`,
+		Multiplex: client.SubscriptionMultiplexRegisteredNamespace(tableName, client.Namespacemicrosoft_compute),
+		Transform: transformers.TransformWithStruct(&armcompute.VirtualMachineAssessPatchesResult{}, transformers.WithPrimaryKeys("AssessmentActivityID")),
+		Columns:   schema.ColumnList{client.SubscriptionID},
```

Comment:
Again, no reference to the actual VM id. `assessment_activity_id` is a uuid and not a resource id, the only way to match/find out which VM it belongs to is to use `_cq_parent_id` unless we add a `ParentColumnResolver("id")` here.